resources/js/app.js

```diff
@@ -10,7 +10,9 @@ import '~/plugins'
 import '~/components'
 
 import '../sass/app.scss'
+import CountryFlag from 'vue-country-flag'
 
+Vue.component('country-flag', CountryFlag)
```

Comment:
Please don't register this globally here (making the whole project slower to load, but instead, lazy-load it on the phone input component

Reply:
Removed